test/content-length.test.js

```diff
@@ -11,7 +11,7 @@ test('default 413 with bodyLimit option', t => {
     bodyLimit: 10
   })
 
-  fastify.post('/', function (req, reply) {
+  fastify.post('/', function (_req, reply) {
```

Comment:
Why have all of these parameters been renamed?

Reply:
I suppose that it is because they are not used in the function.

Reply:
sorry eslinter do it for me

Reply:
Please use the linter configured for the project and revert these unnecessary changes.